redbeams/src/main/java/com/sequenceiq/redbeams/service/UserGeneratorService.java

```diff
@@ -23,4 +23,8 @@ public String generateUserName() {
     public String generatePassword() {
         return UUID.randomUUID().toString();
     }
+
+    public String generatePassword(int maxLength) {
+        return UUID.randomUUID().toString().substring(1, maxLength);
```

Comment:
Should be 0, not 1. But see my overall comment on a better way to drop characters.

Reply:
Yes, I revised the implementation.